test/model/simple_feedforward/test_model.py

```diff
@@ -30,8 +29,11 @@ def hyperparameters():
 
 
 @pytest.mark.parametrize("hybridize", [True, False])
-def test_accuracy(accuracy_test, hyperparameters, hybridize):
-    hyperparameters.update(num_batches_per_epoch=200, hybridize=hybridize)
+@pytest.mark.parametrize("sampling", [True, False])
+def test_accuracy(accuracy_test, hyperparameters, hybridize, sampling):
+    hyperparameters.update(
+        num_batches_per_epoch=200, hybridize=hybridize, sampling=sampling
```

Comment:
hey @YuntianYeAWS this was my bad, apologies: you need to use `distr_output=GaussianOutput()` here, just as you had it before, because the default option (which uses student-T) does not provide access to quantiles, which are used in the test assertions.